### ISSUE TITLE
Copying Portable to UNC path breaks hostnames

### ISSUE THREAD

**Issue by Zewwy** [mRemoteNG/mRemoteNG#1322]
## Expected Behavior
Download MremoteNG portable.
Configure to your hearts desire, set password on config to store passwords.
Move/Copy all files to a new path, in this case a UNC share.
Open MremoteNG, get password request, enter password, then RDP into servers

## Current Behavior
Download MremoteNG portable.
Configure to your hearts desire, set password on config to store passwords.
Move/Copy all files to a new path, in this case a UNC share.
Open MremoteNG, get password, enter password, then:
RDP Disconnect! 260 remote desktop can't find the computer.

## Possible Solution
Not sure.. i checked all computers are inheriting settings from parent folder where the domain string is populated.

## Steps to Reproduce (for bugs)
1) Download MremoteNG portable.
2) Configure to your hearts desire, set inheritance, set password on config to store passwords.
3) Move/Copy all files to a new path, in this case a UNC share.
4) Open MremoteNG, get password, enter password, then:
5) RDP Disconnect! 260 remote desktop can't find the computer.

## Context
I wanted MRemoteNG on my network share to be used from multiple machines, without re-configuring.
The computers paths should remain unchanged...

## Your Environment
Windows 10 x64 1809

**Comment by farosch** [comment]
@Zewwy I cannot reproduce this. Running mrng off a unc path works fine for me.

I have a little trouble understanding step 4 and 5.
As of my understanding you have a password protected connection file. After starting mrng it will prompt you for your password and as soon as you are confirming your password you are getting an RDP error message?
Also, what do you mean by "the computer paths should remain unchanged"?

**Comment by Zewwy** [comment]
Did you follow the steps exactly?

1) Download MremoteNG portable.
2) Configure to your hearts desire, set inheritance, set password on config to store passwords.
3) Move/Copy all files to a new path, in this case a UNC share.
4) Open MremoteNG, get password (cause you set a password on the config file which stores your passwords for the RDP session, thus you are required to enter a password before MremoteNG even starts, enter password, then (attempt an RDP session which you had already configured before moving the files to a UNC path):
5) you get a - RDP Disconnect! 260 remote desktop can't find the computer.

Step 1) Let me reproduce this again, grabbing latest [MremoteNG Portable](https://github.com/mRemoteNG/mRemoteNG/releases/download/v1.76.15/mRemoteNG-Portable-1.76.15.31330.zip)...

Step 2) Extract to a local dir (C:\temp\mrng)

Step 3) Run MremoteNG, configure a couple RDP servers. Save hostname, domain, useranem, and password, connect to RDP server just fine. [Set password on config](https://serverfault.com/questions/842771/mremoteng-how-can-i-set-a-password-on-a-connection-file). (Click Connections node -> Password protect = yes). Save the Connection file.Now when you open MremoteNG it will ask you for the password you set before opening your connection file. Connecting to your existing configured RDP servers connect just fine. Now simply copy the entire MremoteNG folder (C:\temp\mrng) to a UNC path (\\server\my\share\mrng)

Step 4) Now open MremoteNG.exe from the UNC path you just copied it to (\\server\my\share\mrng/MremoteNG.exe). You will get your password prompt just like before as its password protected your logins. Enter your password. MremoteNG will open like normal. However as soon as you atempt to conect to any of your previosuly conect RDP session configurations you get the error!

Step 5) Get Error... [see imgur picture](https://i.imgur.com/ct9SFIs.png)!

If I change the hostname to the servers direct IP I get a different error as seen at the bottom: A interal error occured.

Please fix!

**Comment by farosch** [comment]
@Zewwy I have followed your steps exactly and still cannot reproduce it.

**Comment by farosch** [comment]
Are you maybe using a different client to run mrng from your UNC path as the client you were creating the connections file with?

**Comment by Zewwy** [comment]
I did copy the same folder of files to a USB stick and I did not have this issue, it worked great.

The only other thing I can think of is that the UNC path is secured via Windows Auth, but is mounted automatically via a GPO. So not sure if alllll that makes a difference. Since I'm now just using the secure USB stick instead, this hasn't been an issue for me since. I guess if no one else can reproduce it then it can simply be closed.

Thanks for getting back and trying.
